### PR TITLE
cmd: Set rpc flag defaults

### PR DESCRIPTION
**Pull request by renaynay**
This PR adds flag defaults for rpc addr and port.

### CODE REVIEW

File: cmd/flags_rpc.go [hunk 1]
```diff
@@ -18,12 +18,12 @@ func RPCFlags() *flag.FlagSet {
...
 	flags.String(
 		addrFlag,
-		"",
+		"0.0.0.0",
 		"Set a custom RPC listen address (default: localhost)",
 	)
 	flags.String(
 		portFlag,
-		"",
+		"26658",
```
Comment (tzdybal) [inline cmd/flags_rpc.go:26]: Value is hardcoded in 2 places - here and in https://github.com/celestiaorg/celestia-node/blob/62697fd56ec2b732b3a43afaf9bcad61d139263f/node/rpc/config.go#L12

Comment (tzdybal) [reply cmd/flags_rpc.go:26]: `DefaultConfig()` can be reused here, as a source of "defaults".

Comment (renaynay) [reply cmd/flags_rpc.go]: https://github.com/celestiaorg/celestia-node/pull/696/commits/05163a55c4196b1db89e6f5ea39298691e179fd5

File: cmd/flags_rpc.go [hunk 2]
```diff
@@ -18,12 +19,12 @@ func RPCFlags() *flag.FlagSet {
 
 	flags.String(
 		addrFlag,
-		"",
+		rpc.DefaultConfig().Address,
 		"Set a custom RPC listen address (default: localhost)",
```
Comment (adlerjohn) [inline cmd/flags_rpc.go:23]: Can `rpc.DefaultConfig().Address` not be used to replace `localhost` in this string?

Comment (Wondertan) [reply cmd/flags_rpc.go:23]: +1
I don't think importing config from flags is a good idea.

**Review by Wondertan — CHANGES_REQUESTED** [review]

File: cmd/flags_rpc.go (hunk 2 above)
Comment (adlerjohn) [reply cmd/flags_rpc.go:23]: Oh I meant the string `"Set a custom RPC listen address (default: localhost)"`, not `""`. Are we talking about the same thing?

**Review by adlerjohn — COMMENTED** [review]
Not sure about using writing two literals 3 times each now!

**Comment by Wondertan** [comment]
@renaynay, what was the problem that this PR fixes? Why default config is not enough?

**Comment by Wondertan** [comment]
@renaynay, what was the problem that this PR fixes? Why default config is not enough?

The problem with this approach is that changing the config in the folder won't affect the node as flags have a higher priority than config. The default value for the flag will always make it as it has some value and produce options overriding the loaded config on the node. So after this PR is merged, we will have another issue.

**Comment by Wondertan** [comment]
@renaynay, closing this one until we find a better solution. We need an issue, though. Would be good if you or @tzdybal can explain it

**Comment by renaynay** [comment]
@Wondertan https://github.com/celestiaorg/celestia-node/issues/718

**Pull request closed without merging**